src/Microsoft.SymbolStore.Client/SymbolLocator.cs

```diff
@@ -0,0 +1,162 @@
+﻿using FileFormats.PDB;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FileFormats;
+
+namespace Microsoft.SymbolStore.Client
+{
+    public sealed class SymbolLocator
+    {
+        private static string[] s_microsoftSymbolServerUrls = new string[] { "http://msdl.microsoft.com/download/symbols", "https://nuget.smbsrc.net", "http://referencesource.microsoft.com/symbols", "https://dotnet.myget.org/F/dotnet-core/symbols" };
+        private static ISymbolServer[] s_microsoftSymbolServers = s_microsoftSymbolServerUrls.Select(url => new WindowsSymbolSever(url)).Cast<ISymbolServer>().ToArray();
+
```

Comment:
(Its late, needed to stop here for the night - I'll try to finish reviewing as soon as I can)